internal/config/identity/openid/jwt.go

```diff
@@ -140,6 +142,16 @@ const (
 	azpClaim = "azp"
 )
 
+func replaceClaimsExpiry(timeout string, claims map[string]interface{}) error {
```

Comment:
if we are never returning an error shouldn't we just not return anything?

Reply:
yeah its a good idea.

Reply:
done